### DISCUSSION TITLE
https://{domain}/page?id={page-guid}&startpage=true

### DISCUSSION THREAD

**Discussion by hedronn** [PiranhaCMS/piranha.core#1866] [General] [answered]
Anyone else seeing this format (https://{domain}/page?id={page-guid}&startpage=true) for PiranhaCMS website requests to the homepage URL in logs or browser request monitors? My website using PiranhaCMS appears to use this URL in certain site requests to the homepage which results in a redirect and i cannot see what causes it?!?

**Comment by tidyui** [comment, marked as answer]
This is the internal URL to where the request is rewritten, it should however never result in a redirect unless you have some conflicting middleware installed.

**Reply by hedronn** [reply]
Thanks for the prompt reply. After investigating this a little more, it seems it has to be my sites setup as it occurs when i request my site like the following **https://{domain}** without the www prefix and this then performs a 308 redirect to **https://{domain}/page?id={page-guid}&startpage=true** and resolves to my homepage.  Thanks for the clarification on the rewrite rule, I now know where i need to focus my attention.
